app/rankers/algorithms.py

```diff
@@ -15,7 +15,7 @@
 DEFAULT_ALPHA_PRIOR = 0.02
 DEFAULT_BETA_PRIOR = 1.0
 
-RankableListType = Union[List['SlateModel'], List['RecommendationModel']]
+RankableListType = Union[List['SlateConfigModel'], List['RecommendationModel']]
```

Comment:
For future reference: this is an unrelated bug to this PR. We were using the wrong type here, but no ranker using `RankableListType` was used for slates, so this did not have any impact yet.

Reply:
Gotcha. Thanks for flagging; I would have been confused :)